compact_str/src/features/sqlx/mysql.rs

```diff
@@ -0,0 +1,54 @@
+#![cfg_attr(docsrs, doc(cfg(feature = "sqlx-mysql")))]
+
+use alloc::string::String;
+use alloc::vec::Vec;
+
+use sqlx::encode::IsNull;
+use sqlx::error::BoxDynError;
+use sqlx::mysql::MySqlValueRef;
+use sqlx::{
+    Database,
+    Decode,
+    Encode,
+    MySql,
+    Type,
+    Value,
+    ValueRef,
+};
+
+use crate::{
+    CompactString,
+    ToCompactString,
+};
+
+impl Type<MySql> for CompactString {
+    #[inline]
+    fn type_info() -> <MySql as Database>::TypeInfo {
+        <String as Type<MySql>>::type_info()
+    }
+}
+
+impl<'r> Decode<'r, MySql> for CompactString {
+    fn decode(value: MySqlValueRef<'r>) -> Result<Self, BoxDynError> {
+        let value = value.to_owned();
```

Comment:
Is it possible to avoid this `.to_owned()` call?

Reply:
No needs, it's a reference counted type. Without .to_owned() you cannot do anything with it.